src/steamship/data/block.py

```diff
@@ -207,7 +207,12 @@ def is_image(self):
 
     def is_audio(self):
         """Return whether this is an audio Block."""
-        return self.mime_type in [MimeTypes.MP3, MimeTypes.MP4_AUDIO, MimeTypes.WEBM_AUDIO]
+        return self.mime_type in [
```

Comment:
I think we might want to sneak `OGG` in since Telegram apparently uses that

Reply:
Added OGG_AUDIO and OGG_VIDEO.

Reply:
do we want some sort of catch-all (like `starts_with("audio/")` ?